SDVRPTW/PR01.vrp

```diff
@@ -0,0 +1,234 @@
+NAME: PR01
+COMMENT: Based on Cordeau and Laporte (2001).
+TYPE: SDVRPTW
+EDGE_WEIGHT_TYPE: EUC_2D
+DIMENSION: 49
+VEHICLES: 8
+NODE_COORD_SECTION
+1	23.627	3.963
+2	71.106	31.165
+3	0.549	-73.682
+4	15.039	4.95
+5	45.837	32.874
+6	-27.783	44.08
+7	34.906	20.526
+8	24.31	-7.532
+9	38.629	42.084
+10	7.281	63.531
+11	4.15	59.003
+12	72.937	62.488
+13	-22.791	9.644
+14	30.359	7.294
+15	-4.987	-34.198
+16	12.366	50.69
+17	-5.945	-51.447
+18	5.792	27.319
+19	14.374	37.646
+20	-39.008	-38.196
+21	20.477	-23.016
+22	12.036	-95.831
+23	11.102	-28.54
+24	21.722	-11.42
+25	-30.06	-57.104
+26	61.279	12.836
+27	28.094	-10.779
+28	53.424	18.237
+29	23.126	-24.481
+30	49.591	25.275
+31	18.158	-36.597
+32	8.051	-30.695
+33	46.057	-4.413
+34	4.742	-6.696
+35	-61.401	-81.763
+36	-21.79	33.777
+37	3.766	-49.152
+38	38.556	26.898
+39	19.128	-43.781
+40	27.637	-44.818
+41	6.873	-17.578
+42	11.401	-19.183
+43	-11.151	7.269
+44	72.046	15.271
+45	26.416	-8.6
+46	12.384	86.682
+47	76.721	32.355
+48	32.025	84.741
+49	-14.764	-79.437
+DEMAND_SECTION
+1	0
+2	23
+3	7
+4	24
+5	9
+6	24
+7	17
+8	19
+9	13
+10	6
+11	14
+12	12
+13	15
+14	24
+15	5
+16	16
+17	22
+18	7
+19	17
+20	1
+21	22
+22	5
+23	6
+24	14
+25	1
+26	15
+27	15
+28	7
+29	21
+30	15
+31	2
+32	15
+33	15
+34	9
+35	3
+36	1
+37	5
+38	8
+39	10
+40	12
+41	4
+42	14
+43	17
+44	9
+45	22
+46	7
+47	20
+48	3
+49	2
+SERVICE_TIME_SECTION
+1	0
+2	20
+3	24
+4	12
+5	10
+6	13
+7	4
+8	6
+9	8
+10	7
+11	13
+12	5
+13	17
+14	14
+15	8
+16	5
+17	13
+18	12
+19	2
+20	10
+21	20
+22	16
+23	9
+24	9
+25	16
+26	6
+27	18
+28	1
+29	1
+30	4
+31	14
+32	23
+33	6
+34	6
+35	15
+36	24
+37	8
+38	8
+39	1
+40	10
+41	3
+42	3
+43	23
+44	2
+45	6
+46	17
+47	6
+48	16
+49	3
+TIME_WINDOW_SECTION
+1	0	1000
+2	257	374
+3	325	468
+4	461	576
+5	239	413
+6	281	388
+7	475	628
+8	157	331
+9	70	168
+10	98	237
+11	100	242
+12	253	406
+13	404	571
+14	149	244
+15	325	463
+16	316	459
+17	361	516
+18	139	243
+19	142	259
+20	275	405
+21	121	297
+22	117	266
+23	311	486
+24	321	498
+25	74	205
+26	458	566
+27	284	437
+28	126	275
+29	126	262
+30	314	461
+31	318	464
+32	316	455
+33	430	556
+34	434	539
+35	393	499
+36	323	492
+37	168	343
+38	249	385
+39	125	274
+40	177	323
+41	275	436
+42	165	328
+43	96	228
+44	352	522
+45	435	591
+46	227	364
+47	406	529
+48	91	198
+49	441	604
+CAPACITY_SECTION
+1	100
+2	100
+3	150
+4	150
+5	200
+6	200
+7	250
+8	250
+VEHICLES_MAX_DURATION_SECTION
```

Comment:
Question: does this need to be a section? I don't know these instances, but does this also vary between vehicles?

Reply:
Yeah they are specifications! Changed it.